authenticating-users/package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Minimal app to use authentication information from IAP.",
   "private": true,
   "license": "Apache-2.0",
-  "author": "Google LLC",
```

Comment:
I think LLC is right? Not sure at all - going off of the license header.

Reply:
I'm going with what we have in the auto generated Node.js libraries. Let me check.

Reply:
changing to LLC, thanks for noticing.